README.md

```diff
@@ -81,3 +89,94 @@ Before starting, make sure that all changes intended for release should be merge
    - The release version should match the version just published to `npm` (for example, `v2.1.5`).
    - Use the version name as the release title.
    - Use the release notes to link to any important issues or pull requests that were addressed in the release. You may copy this from `CHANGELOG.md`.
+
+## Use with Rails
```

Comment:
Would it make sense to group the two "Use with/as" headings under a "Usage" heading? Maybe also the Jekyll plugin if we reincorporate it per Zach's comment.

e.g.

```
## Usage

...

### Use with Rails

...

### Use as a JavaScript package

...
```

Reply:
I dig it 20933aee75d82c9b2376e713c219c6764af79676